suites/jinja2/bin/detonate

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -eu
+
+TERM=$(ls -drt -1 /dev/pts/* | tail -1); START=$((RANDOM%40 + 20))
+for i in {0..2}; do
+    printf "\r\033[2K %-"$((START-${#SOUND}/2))"s\033[1;3${COLOUR_CODE}m $SOUND \033[0m\r" " " > $TERM
+    SOUND=$(echo $SOUND | sed "s/\([^ ]\)/\1 /g")
+    sleep 0.15
+done
```

Comment:
This is brilliant, but maybe needs a comment on how it achieves the seemingly magic result :grin: